job/test/test_job_endpoint.py

```diff
@@ -222,6 +222,19 @@ def test_chp_defaults_from_julia(self):
             else:  # Make sure we didn't overwrite user-input
                 self.assertEquals(inputs_chp[key], post["CHP"][key])
 
+    def test_emissions_profile_endpoint(self):
```

Comment:
Can you move this test to the "test_http_endpoints.py" file which is specifically for testing these endpoints that we are exposing through http.jl? I realize that this is still technically testing a job (/dev) view/endpoint, but I think it's useful to split up tests into categories of 1) running REopt with http.jl `reopt` endpoint and 2) non-reopt endpoints, as we start to add more and more tests.